### PR TITLE
add a left_eigenvectors() method

### PR DESCRIPTION
**Pull request by zingale**
add a left_eigenvectors() method that is a wrapper around eigenvectors().
It finds the left eigenvectors of a matrix (l A = lambda l) by taking the transpose of the system,
finding the eigenvectors of A^T, and transposing back.

### CODE REVIEW

File: sympy/matrices/matrices.py [hunk 1]
```diff
@@ -3025,6 +3025,21 @@ def eigenvects(self, **flags):
...
+    def left_eigenvects(self, **flags):
+        """Return list of triples (eigenval, multiplicity, basis).
+           for the left eigenvectors.  Options are the same as
+           for eigenvects()
+        """
+        mat = self
+        left_transpose = mat.transpose().eigenvects(**flags)
+
+        left = []
+
```
Comment (debugger22) [inline sympy/matrices/matrices.py]: Trailing whitepace.

**Comment by debugger22** [comment]
You should add some tests too.

**Comment by zingale** [comment]
Ok, I'll add some tests to test_eigen() -- in doing so I also realized that I wasn't dealing with multiplicity of eigenvalues -- this is fixed in my last commit.

**Comment by debugger22** [comment]
@zingale Please squash all commits into one.

@mrocklin are you okay with this?

**Comment by zingale** [comment]
I seemed to have screwed up the squash (never tried this before).  I thnk that commit 111b777 is the squashed version, but in pushing to my github fork, I had to merge (I guess I should have done a branch?).  Is there an easy way to recover here?

**Comment by debugger22** [comment]
> Is there an easy way to recover here?

Is this not working?

```
$ git rebase -i HEAD~8
```

put `s` in front of previous 7 commits.

**Comment by zingale** [comment]
that doesn't seem to do it for me.

**Comment by smichr** [comment]
Sometimes I deal with this by merging with master then doing a diff wrt master, checking out master, making a temporary branch and applying the diff, then replacing my branch with that:

in this branch....

```
git merge master
git diff master > diff
git checkout master
git checkout -b tmp
git apply diff
git commit -am ok
git commit --amend
git branch -D this_branch
git checkout -b this_branch
git push -f my_repo this_branch
git branch -D tmp
```

**Comment by certik** [comment]
@zingale I am really sorry that this PR got dropped. I took your code, polished it up and created a new PR with it: #9633. So I am closing this one and let's merge #9633 if tests pass.

Thanks again for your work, and next time, please do not hesitate to ping me if any of your code gets stuck. I'll personally make sure it gets in.

**Comment by certik** [comment]
#9633 was merged, closing this one. Thanks @zingale for the contribution and if you have more things to be added to SymPy, please let me know, I'll help out with it.

**Pull request closed without merging**